CHANGELOG.md

```diff
@@ -27,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Full support for SGX KSS (Key Separation and Sharing) including
   - FamilyID and ExtendedProductionID in enclave configuration file. Refer to [Build and Sign an Enclave](docs/GettingStartedDocs/buildandsign.md) for more information.
   - config_id and config_svn at enclave loading time. Refer to [Open Enclave Init-time Configuration Interface](docs/DesignDocs/InitTimeConfigurationInterface.md) for more information.
+- Add the support for SGX quote verification collateral version 3 with the CRL in DER format by default. Refer to [Get Quote Verification Collateral](https://download.01.org/intel-sgx/sgx-dcap/1.10/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf) section 3.3.1.5.
```

Comment:
This doesn't match the title of this PR.
That is, the PR title says 0.16.1 but the change is is to the 0.16.0 log (see line 13).

I'm guessing that's not what was intended and this PR should instead create a v0.16.1 log section.

Reply:
Updated, thanks!